sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -184,7 +180,7 @@ async def list_ownership(
                 - `consumer_group` (str): The name of the consumer group the ownership are associated with.
                 - `partition_id` (str): The partition ID which the checkpoint is created for.
                 - `owner_id` (str): A UUID representing the current owner of this partition.
-                - `last_modified_time` (UTC datetime.datetime): The last time this ownership was claimed.
+                - `last_modified_time` (float): The last time this ownership was claimed as a timestamp.
```

Comment:
just for confirm this is an error for the type hint? in implementation it's always returning the float instead of the datetime.datetime?

Reply:
yep! only changing the docstring since the returned `last_modified_time` value is float